src/style.css

```
@@ -1 +1,54 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:first-child {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars__star:nth-child(-n + 3) {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
```

Too much duplication here
1. There's no need to set width and height for each style, once it is already set for the `stars__star`
2. You can style `.stars--1..5 > .stars__star:nth-child...` with a single expression

```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  margin-right: 4px;
}

.stars--1 > .stars__star:first-child,
.stars--2 > .stars__star:nth-child(-n + 2),
.stars--3 > .stars__star:nth-child(-n + 3) ,
.stars--4 > .stars__star:nth-child(-n + 4) ,
.stars--5 > .stars__star:nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
  background-repeat: no-repeat;
  margin-right: 4px;
}
```